examples/demo_dlmbl/solution.py

```diff
@@ -128,9 +158,9 @@
 
```

Comment:
I now suggest that we ask students to open tensorboard in browser. Scrolling up to look at tensorboard is incovenient. It also doesn't show up in vscode, which we plan to use this year.

Reply:
you mean launching it using the vscode extension or the browser right? Is there a preference?

Reply:
I added instructions on how to use VSCode Ports to output to local web browser.